lib/src/metta/runner/stdlib.rs

```diff
@@ -708,6 +708,65 @@ impl Grounded for PrintlnOp {
     }
 }
 
+// Implement trace! built-in.
```

Comment:
I'm actually getting errors when I replace `//` by `///` such as
```
error: expected one of `(`, `[`, or `{`, found `"Here?"`
 --> src/metta/runner/stdlib.rs:718:10
  |
3 | !(trace! "Here?" 42)
  |          ^^^^^^^ expected one of `(`, `[`, or `{`
```
But I'll investigate tomorrow.

Reply:
It should be  because you have the back-quotes  in the comments. Rust interprets them as examples which should be built and run during the testing stage. As in your case they are examples of the MeTTa code rather than the Rust programs the compiler complains.

Reply:
Possible solutions are:
- using a single back-quote instead of the triple back-quotes;
- specifying a language after the triple back-quotes, for example "```metta".

Reply:
Thanks @vsbogd!